packages/js/src/MatomoTracker.ts

```diff
@@ -54,7 +54,7 @@ class MatomoTracker {
       document.querySelectorAll<HTMLElement>('[data-matomo-event="click"]'),
     )
     if (elements.length) {
-      elements.forEach(element => {
+      elements.forEach((element: HTMLElement) => {
```

Comment:
Why do we need to add a type here? Is TypeScript not smart enough to infer the type?

Reply:
true! can remove it now, but at some point it started to bitch about it...